packages/quantic/cypress/e2e/default-2/recent-results-list/recent-results-list.cypress.ts

```diff
@@ -45,10 +42,6 @@ describe('quantic-recent-results-list', () => {
   });
 
   describe('with default options', () => {
-    beforeEach(() => {
-      clearLocalStorage();
-    });
-
```

Comment:
@erocheleau , we do have testIsolation: true with cypress 13, so essentially LS is cleared between tests.